security/file_integrity_operator/file-integrity-operator-release-notes.adoc

```diff
@@ -13,6 +13,28 @@ These release notes track the development of the File Integrity Operator in the
 
 For an overview of the File Integrity Operator, see xref:../../security/file_integrity_operator/file-integrity-operator-understanding.adoc#understanding-file-integrity-operator[Understanding the File Integrity Operator].
 
+[id="file-integrity-operator-release-notes-0-1-31"]
+== OpenShift File Integrity Operator 0.1.31
+
+The following advisory is available for the OpenShift File Integrity Operator 0.1.31:
+
+* link:https://access.redhat.com/errata/RHBA-2022:103760[RHBA-2022:103760 OpenShift File Integrity Operator Bug Fix and Enhancement Update]
+
+[id="file-integrity-operator-0-1-31-new-features-and-enhancements"]
```

Comment:
This features/enhancement section  is not needed as it is documented in 0.1.30.

Reply:
Done! :)